DigitalLearningSolutions.Data/Models/TutorialContent/TutorialContent.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace DigitalLearningSolutions.Data.Models.TutorialContent
+{
+    public class TutorialContent
+    {
+        public int Id { get; }
+        public int CandidateID { get; }
+        public int SectionID { get; }
+        public int CustomisationID { get; }

```

Comment:
Do we need the candidate id, section id and customisation id? 🙂

Reply:
Probably not, all of this information could be extracted from the route variables. Refactored in 53f828d 🙂